content/notifications/explanation/address-lookup/_index.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Address Lookup
+description: "The principles of address lookup explained. Address lookup is one of the benefits of using Altinn Notifications. 
+However, understanding how it works and which registries are used in various situations is not easy. 
+in this article, we dive deep into the principles of address lookup, explaining the process for different types of notifications."
+weight: 20
+---
+
+
+In relation to address lookup, we categorize notifications into five distinct types:
+
+1. **Notification to Known Email or Mobile Number**
+2. **Notification to National Identity Number without Resource Affiliation**
+3. **Notification to Organization Number without Resource Affiliation**
+4. **Notification to National Identity Number with Resource Affiliation**
+5. **Notification to Organization Number with Resource Affiliation**
```

Comment:
```suggestion
1. **Notification to provided email or mobile number**
2. **Stand alone notification to national identity number**
3. **Stand alone notification to organization number**
4. **Notification associated with a resource to national identity number**
5. **Notification associated with a resource to organization number**
```

Reply:
Yup. Adapted to use you suggested names.